### DISCUSSION TITLE
Accessing raw audio buffers

### DISCUSSION THREAD

**Discussion by amazonparrot** [pine64/OpenPineBuds#97] [General]
I found that we can easily modify pcm audio buffer in process_downlink_bt_voices_frames and process_uplink_bt_voice_frames in apps/audioplayers, I hope this would be helpful to anyone new to this code.

**Comment by Uliz** [comment]
Can you please share more details or example code? I'm trying to modify the data but I'm not sure I'm doing it right.
For example - just for the sake of seeing that I'm able to modify anything, I hardcoded a certain byte to override the entire buffer in process_uplink_bt_voice_frames. For some reason when I record this audio with my phone, it doesn't seem to change at all. I'm recording raw WAV pcm.
